### ISSUE TITLE
Convert FMPPart crash on world load

### ISSUE THREAD

**Issue by flannerby** [Qmunity/BluePower#351]
After running the server just as it gets to Dimension ID -100 (Deep Dark) the server pauses for approximately five seconds, then crashes with the resulting errors. Every log and crash report is located in the link including an OpenEye report. If this is an Extra Utilities error I apologize for the inconvenience.

https://www.dropbox.com/sh/5ude0kqv62ywgco/AADbmzu6dZ9OyiS-Zju0kIAga?dl=0

Edit: There is a zip for the lazy.

**Comment by abalabahaha** [comment]
Do you have ForgeMultipart installed?

**Comment by flannerby** [comment]
I do not have it installed which is why I got this crash I believe. I have more information now however, in an area of the world there was a multipart block(s) and blue power was attempting to convert them and was crashing. Using an nbt editor I found the chunks loaded them with fmp removed and the blocks were deleted. Then server wouldn't crash on startup. Still some sort of issue with blue power trying to convert existing fmp blocks however.
